recipes/easyloggingpp/all/test_package/test_package.cpp

```diff
@@ -0,0 +1,16 @@
+//#define ELPP_FEATURE_CRASH_LOG
```

Comment:
Please minimize the amount of commented-out dead code.

Reply:
Thanks, @madebr, for the review. I had left this out because I wanted to test the optionally enabled features when built. I plan to remove the commented code unless you can suggest a technique of testing the optional features if they are enabled in the recipe being built.

Reply:
worked like a charm! thanks @madebr